pom.xml

```diff
@@ -57,7 +57,7 @@
         <spring.security.version>4.0.3.RELEASE</spring.security.version>
         <spring.version>4.2.2.RELEASE</spring.version>
         <servlet.version>3.0.1</servlet.version>
-        <pac4j.version>1.8.7</pac4j.version>
+        <pac4j.version>1.8.8</pac4j.version>
```

Comment:
It works locally with that in Travis it didn't which is why it is what  it is now